app/src/debug/res/layout/contactpatient_callpatient.xml

```diff
@@ -7,8 +7,8 @@
   android:clipChildren="false"
   android:clipToPadding="false"
   tools:background="?attr/colorSurface"
-  tools:context="org.simple.clinic.contactpatient.views.CallPatientView"
-  tools:parentTag="org.simple.clinic.contactpatient.views.CallPatientView">
+  tools:context="org.simple.clinic.contactpatient.views.CallPatientView_Old"
```

Comment:
We are setting the context and parent tag of the old call patient view to new one in layout?

Reply:
urgh. Changing that. Altho, I wonder why this wasn't caught while testing the layout.